lib/main.dart

```diff
@@ -1,111 +1,54 @@
 import 'package:flutter/material.dart';
+import 'package:lend_a_hand/services/auth_service.dart';
+import 'package:logger/logger.dart';
+import 'package:provider/provider.dart';
 
-void main() => runApp(MyApp());
+import 'home_page.dart';
+import 'login_page.dart';
+
+void main() =>
+    runApp(ChangeNotifierProvider<AuthService>(
+        child: MyApp(),
```

Comment:
Introduce a router with initial route set to the login_page.

Reply:
Done. Using a different approach but should work all the same